### PR TITLE
bs4 home verifications

### PR DESCRIPTION
**Pull request by charlienparker**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/187141984

### CODE REVIEW

**Review by ATBull81 — APPROVED** [review]
Everything looks good, lgtm

File: app/controllers/insured/families_controller.rb [hunk 1]
```diff
@@ -698,6 +698,8 @@ def resolve_layout
       EnrollRegistry.feature_enabled?(:bs4_consumer_flow) ? "progress" : "application"
     when "purchase"
       "application"
+    when "verification"
+      return "progress" if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
```
Comment (kristinmerbach) [inline app/controllers/insured/families_controller.rb:702]: need to add the ternary, need to specify which layout it uses if it isn't enabled

**Comment by kristinmerbach** [comment]
There is a color contrast WAVE error with the Pending badges

File: app/javascript/css/main.scss [hunk 2]
```diff
@@ -319,6 +319,11 @@ ul.list-right-section {
     background-color: var(--warning-status-tint);
     border: 1px solid var(--warning-border);
   }
+
```
Comment (kristinmerbach) [inline app/javascript/css/main.scss:322]: also need to add in an info badge. need to confirm with alex r, but something like:
`  &.badge-info {
    background-color: var(--info-tint);
    border: 1px solid var(--info-color);
  }`

**Comment by kristinmerbach** [comment]
right now nonw of the action dropdowns work - even if they aren't styled the functionality needs to remain.

**Review by kristinmerbach — DISMISSED** [review]
Overall looks good, but i flagged a few things that need to be corrected.

**Comment by charlienparker** [comment]
> right now nonw of the action dropdowns work - even if they aren't styled the functionality needs to remain.

fixed

**Review by ATBull81 — APPROVED** [review]

**Review by kristinmerbach — APPROVED** [review]
I pushed up a few minor changes, but looks good!

**Pull request merged**